docs/project/changelog.md

```diff
@@ -95,14 +86,38 @@ myst:
   JSON (lists and dicts) and JavaScript JSON (Arrays and Objects).
   {pr}`4666`
 
-- {{ Breaking }} The experimental `callSyncifying` method was renamed to
-  `callPromising`.
-  {pr}`4608`
+- {{ Enhancement }} Added a new `callRelaxed` to PyProxies of callables that
+  discards extra arguments rather than raising a `TypeError``.
+{pr}`4392`
 
-- {{ Enhancement }} A new `callWithOptions` method was added to PyProxies of a
-  callable.
+- {{ Enhancement }} A new `callWithOptions` method was added to PyProxies of
+  callables.
   {pr}`4608`
 
+### Build
+
+- {{ Fix }} pyodide-build now use response file when passing list of exported symbols to `emcc`.
+  This fixes "Argument list too long" error.
+  {pr}`4717``
+
+- {{ Fix }} Pass through `-E` (command mode) arguments in CMake wrapper
+  {pr}`4705`.
+
+- {{ Fix }} Fix exception handling in dynamic linking of int64 functions
+  {pr}`4698`.
+
+- {{ Breaking }} `pyodide-build` entrypoint is removed in favor of `pyodide`.
```

Comment:
Should the package be renamed `pyodide`? Now you `pip install pyodide-build` to get `pyodide`, and the command I would use is `pipx run --spec pyodide-build pyodide ...`

Reply:
Also, is there a `pyodide` entry point to replace it? I don't see it in pyproject.toml?

Reply:
I think we were concerned that people will think that `pip install pyodide` is getting the main Pyodide runtime, hence the warning about not being able to install Pyodide via pip. But we should at least tell people that they might want `pyodide-build` there.